__tests__/endpoint.test.js

```diff
@@ -432,4 +432,24 @@ describe('DELETE/api/comments/:comment_id', () => {
           expect(result.body.msg).toBe("No comment found : 444444")
       });
   })
+  })
+
+ describe('GET/api/users', () => {
+    test('responds with status code 200 and a array of object with the correct properties', () => {
+      return request(app).get('/api/users')
+.expect(200).then((response)=>{
+
+          expect(Array.isArray(response.body.users)).toBe(true)
```

Comment:
It's not necessary to check if this is an array, because you are using .forEach() and it only works with arrays, and this test would fail if it wasn't an array.